rootfs/scheduler/__init__.py

```diff
@@ -1053,25 +1047,39 @@ def get_secrets(self, namespace, **kwargs):
 
         return response
 
-    def create_secret(self, namespace, name, data, secret_type='Opaque', labels={}):
+    def _build_secret_manifest(self, namespace, name, data, secret_type='Opaque', labels=None):
```

Comment:
Why is the change to `labels` to a NoneType required? Why not keep it as an empty dictionary?